### ISSUE TITLE
"On the fly" services 

### ISSUE THREAD

**Issue by itsjustk** [bastienwirtz/homer#429]
Hi there! 

Thanks for the great project, I was wondering, is it possible to add your own custom services in the assets folder, or do they have to be built in to the main project? I'm using the docker the official docker image so I don't want to have to readd everytime there's an update. 

Thanks for your time :)

**Comment by Roundaround** [comment]
At the moment it's not possible, but we've actually been chatting about how we could implement a plugin-style system that would grab services from the assets folder. I wouldn't get too excited though because there are several other things we're tackling first and even when we do get to it we need to figure out the technical side of a plugin system (particularly how to facilitate development with IDE/build script support).

Long story short, this will be possible _eventually_, but don't expect it any time soon unfortunately
